creating-campaign/stripe-payment-feature/stripe.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: 'Stripe Connected Payments'
+---
+
+<Note>To enable the Stripe feature and connect your account to Moongate's Stripe account, please contact Moongate for permission.</Note>
+
+## What is Stripe Connect?
+> Stripe Connect is a feature that allows your payments to be sent directly to your account while Moongate handles the associated fees.
+>
+> Imagine you are an event organizer using Moongate to host and manage your events. You want to sell tickets online and receive payments from attendees. Instead of setting up your own payment processing system, you can use Stripe Connect.
+>
+> **Moongate:** Acts as the ticketing platform and integrates with Stripe Connect to facilitate transactions.
```

Comment:
We should change this to 
**User:** Will purchase the ticket from Moongate and pay the price of ticket and platform fees